src/Apps/Legion/Components/WizardShell/WizardShellMockData.ts

```diff
@@ -276,35 +133,35 @@ const setDairyData = (data: IWizardDataContextState) => {
     // PROPERTIES
     const propTemp = getProperty({
         friendlyName: 'Temperature',
-        sourcePropId: 'Temp'
+        sourcePropName: 'Temp'
```

Comment:
how come we renamed this? I think it was id in the class diagram Matt gave us. 🤔

Reply:
It should have been sourcePropName in the first place:
![image](https://user-images.githubusercontent.com/45217314/236049347-9a086eff-bef2-43f4-9fca-f791f7e9b529.png)